src/lib/autoinstall/widgets/storage/disk_usage.rb

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm/common_widgets"
+require "autoinstall/widgets/editable_combo_box"
+
+module Y2Autoinstallation
+  module Widgets
+    module Storage
+      # Widget to determine how the disk will be used
+      #
+      # It corresponds to the `use` element in the profile.
+      class DiskUsage < CWM::ComboBox
+        extend Yast::I18n
+        include EditableComboBox
+
+        def initialize
+          textdomain "autoinst"
+          super
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          _("Reuse")
+        end
+
+        # @macro seeAbstractWidget
+        def opt
+          [:editable]
```

Comment:
what are other options for that reuse? I think it also really needs help to explain other possible values and maybe even some validation.

Reply:
There is a comment in the description of the class. This widget corresponds to the [use element](https://documentation.suse.com/sles/15-SP1/single-html/SLES-autoyast/#CreateProfile-Partitioning-config).